zeppelin-web/src/app/spell/spell-result.js

```diff
@@ -18,6 +18,7 @@
 export const DefaultDisplayType = {
   ELEMENT: 'ELEMENT',
   TABLE: 'TABLE',
+  NETWORK: 'NETWORK',
   HTML: 'HTML',
   ANGULAR: 'ANGULAR',
   TEXT: 'TEXT',
```

Comment:
should `NETWORK` be added to the end?
also, should `%network` be in the DefaultDisplayMagic list on line L27 below?